lib/secure/simul/livings.c

```diff
@@ -23,17 +23,13 @@ object find_npc(string name) {
 }
 
 object find_living(string name) {
-    object ob;
-    if(!(ob=LIVINGSD->get_living(name, 0)))
-        ob=LIVINGSD->get_living(name, 1);
-    return ob;
+    return LIVINGSD->get_living(name, 0)) || LIVINGSD->get_living(name, 1);
```

Comment:
nice simplification here. split to two lines though

Reply:
Done.